### DISCUSSION TITLE
Making a mini area chart.

### DISCUSSION THREAD

**Discussion by AzadCoder** [nhn/tui.chart#650] [Q&A]
I'm trying to make a mini area chart without any label or title. just the chart. But I don't know How to remove extra stuff.
My chart looks like this.
![toast-ui-chart](https://user-images.githubusercontent.com/77876948/114260845-83f2ef80-99ec-11eb-95f0-366423ddae3c.png)
 By the way, I'm using the library in an angular project.

**Comment by jung-han** [comment]
![image](https://user-images.githubusercontent.com/35371660/114348903-26080880-9ba2-11eb-8ce4-95bf9924a65c.png)

How about reducing the size after removing elements that appear through the visible option by referring to this guide?

```js
const options = {
  exportMenu: {
    visible: false,
  },
  legend: {
    visible: false,
  },
  // ...
}

```

* https://github.com/nhn/tui.chart/blob/main/docs/en/common-chart-options.md
* https://github.com/nhn/tui.chart/blob/main/docs/en/common-layout-options.md
* https://github.com/nhn/tui.chart/blob/main/docs/en/common-legend.md
